2019-pydays-vienna/typos-correction-with-lookout.md

```diff
@@ -0,0 +1,36 @@
+# Title of the Talk
+
+**Updates**:  https://github.com/src-d/conferences/issues/133
+
+**Sent on**:  -
+
+**Status**:   on preparation
+
+**Author**:   Irina
+
+**Bio**: Junior Machine Learning Engineer at source{d}. Persue Master's degree of Computer Science and Data Analysis at MIPT (Moscow
```

Comment:
Irina is a Junior Machine Learning Engineer at source{d} (sourced.tech). She pursues a Master's degree in Computer Science and Data Analysis at Moscow Institute of Physics and Technology and is studying at Yandex School of Data Analysis.